Actions/AL-Go-Helper.ps1

```diff
@@ -634,6 +634,7 @@ function ReadSettings {
         "installTestLibraries"                          = $false
         "installPerformanceToolkit"                     = $false
         "enableCodeCop"                                 = $false
+        "checkForNewWarnings"                           = $false
```

Comment:
blockNewWarnings or failOnNewWarnings is better IMO

Reply:
failOnNewWarnings is my favorite